### DISCUSSION TITLE
HPO Terms and IDs in "Case Information"

### DISCUSSION THREAD

**Discussion by alexejknaus** [bihealth/reev#227] [General]
In the function "Case Information" entering HPO IDs doesn't work, comma or semicolon separated lists of HPO IDs do not work either, parts of IDs (e.g. just the number) doesn't work and when entering HPO Terms, often prediction aborts, when more than one term is already entered.
Example trying to add intellectual disability:

![image](https://github.com/bihealth/reev/assets/37909805/73f6e620-7493-43b1-9bd1-961cc78264eb)

Otherwise interesting tool!

**Comment by holtgrewe** [comment]
Thanks for the feedback. There appear to be a number of issues:

- input of numbers is not implemented yet
- the official name of "intellectual disability" is now "cognitive impairment" and we don't provide access to term aliases yet
- spaces somehow don't get through to the backend "cognitive impairment"

**Reply by holtgrewe** [reply]
Intellectual disability can now be found in the next deployed version via our integrated full-text engine.

![image](https://github.com/bihealth/reev/assets/521780/53c81a2f-a270-41a0-bc45-ef5677681ed3)

Similarly, search by HPO identifier works now.

![image](https://github.com/bihealth/reev/assets/521780/ab592416-ae37-4968-9bce-4f8cdce31000)
